xbmc/pvr/channels/PVRChannelGroupInternal.cpp

```diff
@@ -223,7 +229,7 @@ bool CPVRChannelGroupInternal::CreateChannelEpgs(bool bForce /* = false */)
   {
     CSingleLock lock(m_critSection);
     for (auto& groupMemberPair : m_members)
-      CreateChannelEpg(groupMemberPair.second->Channel());
+      groupMemberPair.second->Channel()->CreateEPG();
```

Comment:
Can channel be null here?

Reply:
No, it must not. If it is, we would face a bug which must not be worked around by nullptr check. :-).

Reply:
Cool